src/app/questions/file/file.component.ts

```diff
@@ -113,4 +125,30 @@ export class FileComponent implements ControlValueAccessor, OnInit {
 
   }
 
+  // adding save values to from control
+  private _showSavedAnswers() {
+    if ((this.reviewStatus === 'in progress') && (this.doReview)) {
+      if (!this.innerValue) {
```

Comment:
At this time, the `innerValue` is not initialised for sure, so don't need this condition check
Same for all other files